apps/stack/template.yaml

```diff
@@ -79,12 +80,18 @@ Parameters:
       of this stack, and must therefore be unique within the account.
     Default: ""
     MaxLength: 51
+  ConfigDeliveryBucketName:
```

Comment:
Where do we manage IAM permissions for the forwarder for this pre-existing Config bucket?